pymemcache/client/base.py

```diff
@@ -814,9 +818,10 @@ def _store_cmd(self, name, values, expire, noreply, cas=None):
 
             key = self.check_key(key)
             if self.serializer:
-                data, flags = self.serializer(key, data)
-            else:
-                flags = 0
+                if flags == 0:
```

Comment:
How about:

```python
if self.serializer:
    data, serializer_flags = self.serializer(key, data)
    # Use the serializer's flags when `flags` haven't been specified.
    if flags == 0:
        flags = serializer_flags
```

Reply:
looks good, will change the code base on it.